src/docfx/config/ops/OpsAccessor.cs

```diff
@@ -334,14 +334,20 @@ private static async Task FillOpsToken(HttpRequestMessage request, DocsEnvironme
                 // For development usage
                 try
                 {
-                    var token = (environment ?? DocsEnvironment) switch
+                    var token = EnvironmentVariable.OPSBuildUserToken;
+                    if (string.IsNullOrEmpty(token))
```

Comment:
use `??`

Reply:
seems ?? don't handle the empty string?

Reply:
`EnvironmentVariable.GetValue` internally handles empty string, to make it work with `??`

Reply:
Hah, great, but seem it cannot be used in this case since we need another variable `tokenTask`